EIPS/eip-634.md

```diff
@@ -45,13 +46,15 @@ services must be prefixed with `vnd.`.
 - **url** - a URL
 - **avatar** - a URL to an image used as an avatar or logo
 - **description** - A description of the name
-- **notice** - A notice regarding this name; 
+- **notice** - A notice regarding this name
 - **keywords** - A list of comma-separated keywords, ordered by most significant first; clients that interpresent this field may choose a threshold beyond which to ignore
 
 - **vnd.github** - a GitHub username
-- **vnd.peepeth** - a peepeth username
-- **vnd.twitter** - a twitter username
-
+- **vnd.gitlab** - a GitLab username
+- **vnd.peepeth** - a Peepeth username
+- **vnd.twitter** - a Twitter username
+- **vnd.keybase** - a Keybase username
+- **vnd.reddit** - a Reddit username
 
```

Comment:
```suggestion
- **vnd.telegram** - a Telegram username

```

To address part of https://github.com/ethereum/EIPs/issues/2439#issuecomment-614827905